javascript/javascript2/week1/problemSolvingCardio/findTheShortestWord.js

```diff
@@ -1,42 +1,49 @@
-const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
-
-console.log(
-  danishWords.reduce(function(a, b) {
-    return a.length <= b.length ? a : b;
-  })
-)
-
-let shortestWord = danishWords.reduce(
-  function  (a,b) {
-    if (a.length <= b.length){
-      return a;
-    }  
-  }
-)
+const danishWords = ["bil", "plante", "kaffe", "bog", "planetarium", "ø"];
+
+
+
+// sort array according to string length
+
+const shortesdWord = danishWords.sort(function(a, b){return a.length - b.length});
```

Comment:
Amazing solution, especially since I can tell that you are using the `Array` method `.sort()`. It is always good to let the programming language do the hard work and thinking for us, just like the `for(let x of y)` construction. 🦾